ifStatement.js

```diff
@@ -59,6 +73,13 @@ Steps:
     - If the user is younger than 18, the program should calculate how many years are left until they turn 18 and print the message "You will be eligible to vote in X years", where X is the number of years left.
 ************************************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let userage=18
```

Comment:
The variable userage is assigned to 18. Instead, you should prompt the user to input their age and convert the input to an integer as mentioned in the description.